src/command/backup/backup.c

```diff
@@ -1573,7 +1573,8 @@ backupJobResult(
                     file.checksumPageError = checksumPageError;
                     file.checksumPageErrorList =
                         checksumPageErrorList != NULL ? jsonFromVar(varNewVarLst(checksumPageErrorList)) : NULL;
-                    file.bundleId = bundleId;
+                    // Truncated file is not put in bundle
+                    file.bundleId = copyResult != backupCopyResultTruncate ? bundleId : 0;
```

Comment:
I think we should also change the log message when the file is truncate. Something like:
```
store truncated file pg1/grow-to-block-incr (4B->0B, [PCT])
```
No need for the checksum or bundle info in this case, but knowing the original size before it was truncated might be useful for debugging.

Reply:
done